### PR TITLE
chore(README): add create task to readme

### PR DESCRIPTION
**Pull request by fedealconada**
## Motivation

As part of our end to end release process, we wanted to give more clarity on how Hardhat tasks are created and executed.

## Solution

Include guide on how tasks are created and what dev needs to do to deploy new contracts or upgrade existing ones.

## Implementer Checklist

- [ ]  [FIRST TIME ONLY] Review the [Solcurity](https://github.com/Rari-Capital/solcurity) standard
- [x]  [OPTIONAL] Create a reference implementation (in python or JS)
- [x]  Create a short "Motivation" section for the PR
- [ ]  Write a spec for the feature and put it in the PR description – basic function names and expected state transitions is OK
- [ ]  Check all of the new revert paths with concrete tests
- [ ]  Check all of the new storage slot writes with concrete tests – do not make the passing test case the trivial case!
- [ ]  Go line-by-line through the new code and ensure it has all been covered by concrete tests – since we don’t have a coverage engine for foundry yet, you can imagine how one might check that each different code path is covered
- [ ]  Simplify the implementation and spend some time trying to minimize gas costs
- [ ]  Write fuzz tests for the feature – try everything to break the implementation (is this function monotonically in/decreasing, should it always be less than something else, etc)
- [ ]  Capture bugs discovered during the above steps in concrete tests (regression tests)
- [ ]  Add integration tests – fuzz or concrete tests that test how the new code affects the entire system, either locally or with a mainnet fork
- [ ]  [OPTIONAL] Integrate the feature into the deployment scripts
- [ ]  [OPTIONAL] Add new sanity checks to the deployment scripts
- [ ]  Get the PR reviewed by at least two people

### CODE REVIEW

**Review by abylaw — APPROVED** [review]
LGTM!! Tyty

File: README.md [hunk 1]
```diff
@@ -101,32 +101,91 @@ yarn lint
 yarn fix
 ```
 
-### Deploy
+### Environment
+
+1. Create a local `.env` file in the root directory of this project
+2. Set `ALCHEMY_KEY` to a valid Alchemy API key
+3. Set `MNEMONIC` to a valid seed phrase for deployments
```
Comment (jparklev) [inline README.md:108]: And an etherscan key if we're doing verification!

File: README.md [hunk 2]
```diff
@@ -101,32 +101,91 @@ yarn lint
...
+
+    # Execute hardhat task
+    yarn hardhat 20220517-long-wsteth-adapter --network mainnet
+    ```
+
+    *It is recommended that you first test that your task does what's intended by running it both on a local network and Goerli. Just use `--network hardhat` or `--network goerli` when executing the command.*
+
+4. Add *output* files into task folder:
+
+    After succesfully executing the task, Hardhat would have saved the resulting deployments (contract addresses along their abi, bytecode, metadata...) on `pkg/deployments/deployments`. **Move that folder inside `pkg/deployments/tasks` and rename it to `output`**.
```
Comment (jparklev) [inline README.md]: Inside `pkg/deployments/tasks/[task-name]`, right?

File: README.md [hunk 3]
```diff
@@ -101,32 +101,91 @@ yarn lint
...
+
+    ```bash
+    # Navigate to the `deployments/tasks` folder
+    cd pkg/deployments/tasks
+
+    # Execute hardhat task
+    yarn hardhat 20220517-long-wsteth-adapter --network mainnet
+    ```
+
+    *It is recommended that you first test that your task does what's intended by running it both on a local network and Goerli. Just use `--network hardhat` or `--network goerli` when executing the command.*
```
Comment (jparklev) [inline README.md]: Yea, the hardhat network seems crucial to me then the goerli test is maybe not as important overall

**Review by jparklev — APPROVED** [review]
Nits, but LGTM 🌤️

**Pull request merged**